packages/terra-outline-view-table/src/terra-dev-site/test/outline-view-table/OutlineViewTable.test.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import OutlineTableView from 'terra-outline-table-view';
+import OutlineTableView from 'terra-outline-view-table';
 
 const DefaultTest = () => (
   <OutlineTableView
```

Comment:
Should the changes be done here as well

Reply:
Good catch. I'll update.

Reply:
https://github.com/cerner/terra-core/pull/3692/commits/5d5e261a193d64dec07f1f49f29f8e2ee8baf2c7